Objects/Objects/BuiltElements/Floor.cs

```diff
@@ -95,9 +97,9 @@ namespace Objects.BuiltElements.Archicad
   public sealed class ArchicadFloor : Floor
   {
     // Element base
-    public string? elementType { get; set; } /*APINullabe*/
```

Comment:
@jozseflkiss remind me, what were these `APINullable` comments for? 
And, are Claire's changes to this class correct? I seem to remember you were experimenting with keeping some of these props non-nullable for Archicad -> Speckle, but nullable for Archicad C# -> C++